src/components/PropertyDetailSection.tsx

```diff
@@ -250,6 +250,9 @@ const PropertyDetailSection: FC<PropertyDetailSectionProps> = ({
                 disableCursorAnimation={true}
               ></Pagination>
             </div>
+            <p className="text-center mt-4">
+              {`${((page - 1) * 6) + 1}-${page === pages ? featuresInView.length : (page * 6)} of ${featuresInView.length}`}
```

Comment:
Can we change the "-" to the text "to" instead?

Right now on VoiceOver it reads out as "one six of one hundred" , with the text swap it will be "one to six of one hundred"

Reply:
Yep!